lib/spoom/cli.rb

```diff
@@ -21,17 +17,81 @@ class Main < Thor
 
       map T.unsafe(["--version", "-v"] => :__print_version)
 
+      desc "srb", "Sorbet related commands"
+      subcommand "srb", Spoom::Cli::Srb::Main
+
       desc "bump", "Bump Sorbet sigils from `false` to `true` when no errors"
-      subcommand "bump", Spoom::Cli::Bump
+      option :from,
+        type: :string,
+        default: Spoom::Sorbet::Sigils::STRICTNESS_FALSE,
+        desc: "Change only files from this strictness"
+      option :to,
+        type: :string,
+        default: Spoom::Sorbet::Sigils::STRICTNESS_TRUE,
+        desc: "Change files to this strictness"
+      option :force,
+        type: :boolean,
+        default: false,
+        aliases: :f,
+        desc: "Change strictness without type checking"
+      option :sorbet, type: :string, desc: "Path to custom Sorbet bin"
+      option :dry,
+        type: :boolean,
+        default: false,
+        aliases: :d,
+        desc: "Only display what would happen, do not actually change sigils"
+      option :only,
+        type: :string,
+        default: nil,
+        aliases: :o,
+        desc: "Only change specified list (one file by line)"
+      option :suggest_bump_command,
+        type: :string,
+        desc: "Command to suggest if files can be bumped"
+      option :count_errors,
+        type: :boolean,
+        default: false,
+        desc: "Count the number of errors if all files were bumped"
+      option :sorbet_options, type: :string, default: "", desc: "Pass options to Sorbet"
+      sig { params(directory: String).void }
+      def bump(directory = ".")
+        say_warning("This command is deprecated. Please use `spoom srb bump` instead.")
```

Comment:
People may wonder thew new one is the same or not. Would be better to say it's been 'renamed' or 'moved'? Or we could append, "...instead, which has identical behavior".

Reply:
Do you think the release notes would be enough?

Reply:
Maybe, but I suspect few people read them.